commitlint.config.js

```diff
@@ -0,0 +1,5 @@
+
+module.exports = {
+	extends: ['@commitlint/config-conventional']
+}
+
```

Comment:
Why was this added?

Reply:
Since you don't need my commit tools, see `scripts/install.sh`, I can delete it safely. It provides my git commit hook.